archive/100-theorems-list/73_ascending_descending_sequences.lean

```diff
@@ -0,0 +1,181 @@
+import data.finset
```

Comment:
The doc requirements for stuff in `archive/` are much looser than in mathlib proper, but you should at least have a copyright header + module doc.